.eslintrc.json

```diff
@@ -1,27 +1,26 @@
 {
-    "root": true,
-    "env": {
-        "browser": true,
-        "es2020": true,
-        "node": true,
-        "mocha": true
-    },
-    "extends": [
-        "plugin:prettier/recommended",
-        "plugin:@typescript-eslint/recommended",
-        "prettier/@typescript-eslint",
-        "prettier",
-        "eslint:recommended",
-        "airbnb-typescript"
-    ],
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": 7,
-        "sourceType": "module"
-    },
-    "plugins": [
-        "@typescript-eslint"
-    ],
-    "rules": {
-    }
+  "root": true,
+  "env": {
+    "browser": true,
+    "es2020": true,
+    "node": true,
+    "mocha": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+    "airbnb-base"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 7,
+    "sourceType": "module",
+    "project": "./tsconfig.eslint.json"
+  },
```

Comment:
Deleted `airbnb-typescript` because it's related to `react` formatting,
and added `airbnb-base`.
And also set the `project` field in `parserOptions.project`.
This field is needed for configuring the project path to lint.